gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineIdeaIntegrationTest.groovy

```diff
@@ -86,6 +86,48 @@ class BaselineIdeaIntegrationTest extends AbstractPluginTest {
         rootIpr.contains('<component name="CopyrightManager" default="999_palantir.txt">')
     }
 
+    def 'Idea project has copyright configuration when importing'() {
+        when:
+        buildFile << standardBuildFile
+        System.setProperty("idea.active", "true")
+
+        then:
+        with('idea').build()
```

Comment:
this should work even without running 'idea' (that task is not run when importing from intellij), so `with().build()` would be enough